frontend/src/locales/scripts/en.json5

```diff
@@ -1141,9 +1141,9 @@
     resultCountLabel: {
       creator: {
         audio: {
-          zero: "No audio files by this creator.",
-          count: "{count} audio file by this creator.|{count} audio files by this creator.",
-          countMore: "Over {count} audio files by this creator.",
+          zero: "No audio track files by this creator.",
+          count: "{count} audio track file by this creator.|{count} audio track files by this creator.",
+          countMore: "Over {count} audio track files by this creator.",
```

Comment:
Same note as above^. The use of "track(s)" will eliminate the need for the word "file(s)".